content/docs/start/data-and-model-access.md

```diff
@@ -83,9 +83,9 @@ bring in changes from the data source later, using `dvc update`.
 > `dvc import` downloads from [remote storage](/doc/command-reference/remote).
 
 `.dvc` files created by `dvc import` have special fields, such as the data
-source `repo`, and `path` (under `deps`):
+source `repo` and `path` (under `deps`):
 
-```diff
+```git
 + deps:
 + - path: get-started/data.xml
 +   repo:
```

Comment:
Also, a full fix here (including indentation) would require removing those spaces like:

```suggestion
```git
+deps:
+- path: get-started/data.xml
+  repo:
```
but there are 2 more lines that I can't suggest an edit for from GH.

Reply:
Extracted all these (and more) to #2462 to simplify this PR a little, please review/approve if possible @casperdcl .

Reply:
OK that PR is in, want to merge `master`? 🙂